### DISCUSSION TITLE
Wiki Translation Authorization Request

### DISCUSSION THREAD

**Discussion by Codesire-Deng** [axboe/liburing#960] [General]
Hi @axboe. I hope this message finds you well. I am writing to seek your permission to translate your inspiring wiki post titled ["io_uring and networking in 2023"](https://github.com/axboe/liburing/wiki/io_uring-and-networking-in-2023) into Chinese. I believe that your unique perspectives and thoughtful design deserves a wider audience, particularly among Chinese speakers. 

If you give permission, I will do the translation. The translated article will be shared on my public social media platforms, where it can be read freely without any commercial profit, and you will be fully credited as the original author with your name and original source clearly indicated.

I assure you that I will respect your rights as the owner of the content, and I am open to comply with any conditions or limitations you may have concerning the use of your work. Please inform me at your convenience if you grant me the permission to translate and share your wiki post.

Thank you very much for your time. I look forward to hearing from you soon.

**Comment by axboe** [comment]
Certainly, go ahead! This sounds like a great idea.

**Comment by ywave620** [comment]
Great! Please leave a link when you're done @Codesire-Deng

**Comment by Codesire-Deng** [comment]
The translation has been posted to [zhihu.com](https://zhuanlan.zhihu.com/p/661330392). Thank you again, Axboe, for your wonderful article！ FYI @ywave620 .

**Reply by axboe** [reply]
Thank you!
